Makefile

```diff
@@ -2,7 +2,8 @@ install:
 	pipenv install --dev
 
 check:
-	PIPENV_PYUP_API_KEY="" pipenv check
+	# TODO: 38932 - Explicitly ignore (unused) cryptography package warning for now
```

Comment:
The cryptography package is used.

Reply:
Sorry, I'll update my wording a bit better!